yoti_python_sdk/doc_scan/session/create/filter/document_restrictions_filter.py

```diff
@@ -0,0 +1,154 @@
+from yoti_python_sdk.doc_scan.constants import (
+    DOCUMENT_RESTRICTIONS,
+    INCLUSION_BLACKLIST,
+    INCLUSION_WHITELIST,
+)
+from yoti_python_sdk.utils import YotiSerializable
+from .document_filter import DocumentFilter
+
+
+class DocumentRestriction(YotiSerializable):
+    def __init__(self, country_codes, document_types):
+        self.__country_codes = country_codes
+        self.__document_types = document_types
+
+    @property
+    def country_codes(self):
+        return self.__country_codes
+
+    @property
+    def document_types(self):
+        return self.__document_types
+
+    def to_json(self):
+        return {
+            "country_codes": self.country_codes,
+            "document_types": self.document_types,
+        }
+
+
+class DocumentRestrictionBuilder(object):
+    def __init__(self):
+        self.__country_codes = None
+        self.__document_types = None
+
+    def with_country_codes(self, country_codes):
```

Comment:
```suggestion
    def with_countries(self, country_codes):
```

Perhaps this should match naming in Java? (I think it's close enough to leave it as is)

Reply:
As discussed, we're keeping them as they are